src/modules/helpers/index.ts

```diff
@@ -445,6 +445,49 @@ export class HelpersModule {
     return array[index];
   }
 
+  /**
+   * Returns a weighted random element from the given array. Each element of the array should be an array with two elements: the first is the value, the second is an integer weight.
+   * For example, if there are two elements A and B, with weights 1 and 2 respectively, then the probability of picking A is 1/3 and the probability of picking B is 2/3.
+   *
+   * @template T The type of the entries to pick from.
+   * @param array Array to pick the value from.
+   *
+   * @example
+   * faker.helpers.weightedArrayElement([['sunny', 5], ['rainy', 4],['snowy', 1]]) // 'sunny', 50% of the time, 'rainy' 40% of the time, 'snowy' 10% of the time
```

Comment:
```suggestion
   * faker.helpers.weightedArrayElement([['sunny', 5], ['rainy', 4], ['snowy', 1]]) // 'sunny', 50% of the time, 'rainy' 40% of the time, 'snowy' 10% of the time
```

Or whatever prettier would produce.